src/common.c

```diff
@@ -134,12 +145,19 @@ void Com_Print(const char *fmt, ...) {
 		fputs(msg, stdout);
 		fflush(stdout);
 	}
+
+	// FIXME: not sure how to resolve this one, because
+	// prints happen before command buf is run and whatnot.
+	// developer may never be non-zero if crash happens very early.
```

Comment:
Kind of. It's moreso just noting that there's no way to print -everything- out to the attached debugger because the command buffer that enables debug happens several steps down the chain (if you remove the !debug check, for instance, it will crash).